bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2541,21 +2542,39 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
          Utility::AddToClass( pyclass, "__setitem__", (PyCFunction) VectorBoolSetItem );
       }
 
-      // add array interface
-      if (name.find("<float>") != std::string::npos) {
+      // add array interface for STL vectors
+      auto hash = std::hash<std::string>();
+      const auto hashName = hash(name);
+      if (name.find("ROOT::VecOps::RVec<") != std::string::npos) {
+      } else if (hashName == hash("vector<float>")) {
```

Comment:
Is the hash convenient in this case given the small strings?

Reply:
Fixed.